_posts/2024-04-01-pyos-newsletter-april-2024.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: single
+title: "pyOpenSci Newsletter: April 2024"
+excerpt: "New contributors, a new package, and a few new social campaigns, all to keep you connected with the pyOpenSci community!"
+author: "Jesse Mostipak"
+permalink: /blog/pyos-newsletter-april-2024.html
+header:
+  overlay_color: "#666"
+  overlay_filter: 0.6
+categories:
+  - blog-post
+  - community
+classes: wide
+toc: true
+comments: true
+---
+
+## <i class="fa-solid fa-heart"></i> To 200 and beyond!
+
+This past month, pyOpenSci reached a major milestone: [200 contributors](https://www.pyopensci.org/our-community/index.html)! We have a diverse and vibrant community of Pythonistas, and are thrilled that 200 (and counting) wonderful people have contributed to pyOpenSci. Will you be next? [Learn more about getting involved](https://www.pyopensci.org/volunteer.html).
+
+
+## <i class="fa-solid fa-camera"></i> Launched: editor spotlight
+
+pyOpenSci editors are volunteers who fulfill a critical role within pyOpenSci. Editors lead the open peer review process for three to four Python packages a year, and also weigh in on group editorial decisions. With everything they do for pyOpenSci, we wanted to celebrate them! In March we kicked off our pyOpenSci editor spotlight, where every Wednesday we share a short interview with one of our editors. You can catch up with previous spotlights, and stay tuned for more, on both our [Fosstodon](https://fosstodon.org/@pyOpenSci) and [LinkedIn](https://www.linkedin.com/company/pyopensci) pages.
+
+## <i class="fa-solid fa-computer"></i> PyCon US 2024 Maintainers Summit
+
+We’ve been talking (a lot) about the [PyCon Maintainers Summit](https://us.pycon.org/2024/events/maintainers-summit/) (which is now full, but [you can sign up for the waitlist](https://us.pycon.org/2024/events/maintainers-summit/)), happening this May in Pittsburgh. The Maintainers Summit, included in your [PyCon registration](https://us.pycon.org/2024/accounts/login/?next=/2024/registration/register), is where we come together to discuss and foster best practices on how to develop sustainable projects and nurture thriving communities. And thanks to [Mariatta Wijaya](https://mariatta.ca/), we’ve got an inside look into everything you can expect in [this incredible video](https://www.youtube.com/watch?v=L-Ok_89QJOM) featuring Kara Sowles, Inessa Pawson, and our Executive Director, Leah Wasser!
+
+<a href="https://www.youtube.com/watch?v=L-Ok_89QJOM" target="_blank">
```

Comment:
hm we can do a video embed here. 

this is a example post with the video (we are loosely using minimal mistakes theme but it's hacked)

https://mmistakes.github.io/minimal-mistakes/post%20formats/post-video-youtube/

[here is more](https://mmistakes.github.io/minimal-mistakes/docs/helpers/#youtube) on embedding a video . can you please add a commit that embeds the video properly?

Reply:
ok no problem. i just did some debugging - i saw that you weren't trying to embed the video, you were linking to it . when it didn't render for me i thought it was an embed gone wrong. but in fact, something weird is happening with github/ci here. let's revisit when you submit another pr. thank you for these changes!